prep-javascript-objects/main.js

```diff
@@ -0,0 +1,13 @@
+var person = {
+  firstName: 'changhyun',
+  lastName: 'Yoo',
+  hobby: 'Game'
+};
+console.log("The Person's name is: ", person.firstName);
+console.log(person.lastName);
```

Comment:
Hi Changhrun, you can get the person's full name by concatenating their first and last names

```js
var fullName = person.firstName + ' ' + person.lastName;
```

Reply:
please let me know if i can merge it thanks!